### PR TITLE
event improvement (added internal link to events)

### LINKED ISSUE #1813: Layer5 event website improvment
**Current Behavior**
When an event card is clicked it takes the user straight to an external link for the card

**Desired Behavior**
When the title is clicked we want it to take the user to an internal page that contains a short write up about the event

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by Lezette**
Signed-off-by: Elizabeth <leezeelola@gmail.com>

**Description**
Added an internal link to event cards and fixed CSS for cards component to contain multiple links, instead of adding a link on the header, since there's already a see more button in the card that does what was needed, I fixed the CSS to allow multiple links in the card and added the fields section in the events

This PR fixes #1813

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 2a3f51bf4725b4f8b6b27536f4a416d633b8af49 at: https://608364ee20535d1ae7fd281f--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 2a3f51bf4725b4f8b6b27536f4a416d633b8af49 at: https://6083652c20535d17c7fd2a79--layer5ng.netlify.app

**Comment by leecalcote** [comment]
Quick work, @Lezette. Will you consider updating the behavior to be that if the user clicks anywhere on the card (other than the external arrow icon) they are directed to the respective event page within layer5.io?

**Comment by Lezette** [comment]
Sure! I’ll do that

File: src/sections/Events/index.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ import Button from "../../reusecore/Button";
 
 const Meetups = ({ data, pageContext }) => {
   const [active, setActive] = useState("all");
+  console.log("data: ", data);
```
Comment (Nikhil-Ladha) [inline src/sections/Events/index.js:12]: Remove this

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
Making changes in the main component might break things at other places where the same component is used.
For example, the cards on the `news` page don't work properly anymore.
Anyways, I think the CSS change might not be required because it was coded in a good way previously to consider this case. It was just that the `fields` value was not passed. Have you tried that?

**Comment by Nikhil-Ladha** [comment]
If you check the `/news` page on the live site, it does a similar thing with the same CSS.

**Comment by Lezette** [comment]
Alright I’ll check that, I actually added the field and it didn’t work. I think I’ll check what I’m doing then

**Comment by Lezette** [comment]
> Making changes in the main component might break things at other places where the same component is used.
> For example, the cards on the `news` page don't work properly anymore.
> Anyways, I think the CSS change might not be required because it was coded in a good way previously to consider this case. It was just that the `fields` value was not passed. Have you tried that?

@Nikhil-Ladha I made reverted the changes, but currently, the news page on the live site only directs to one URL even when you click on the 'see more' link. you can check it out. That was why I made changes to the CSS

**Comment by Nikhil-Ladha** [comment]
> > Making changes in the main component might break things at other places where the same component is used.
> > For example, the cards on the `news` page don't work properly anymore.
> > Anyways, I think the CSS change might not be required because it was coded in a good way previously to consider this case. It was just that the `fields` value was not passed. Have you tried that?
> 
> @Nikhil-Ladha I made reverted the changes, but currently, the news page on the live site only directs to one URL even when you click on the 'see more' link. you can check it out. That was why I made changes to the CSS

What do you mean by one URL?
There are 2 links in that card one is embedded to the whole card, that is wherever you click on the card it will redirect you to the news page within the layer5 site, except the 'new_page' icon, which redirects you to a separate page outside Layer5.
I have confirmed the same.

**Comment by Lezette** [comment]
> > > Making changes in the main component might break things at other places where the same component is used.
> > > For example, the cards on the `news` page don't work properly anymore.
> > > Anyways, I think the CSS change might not be required because it was coded in a good way previously to consider this case. It was just that the `fields` value was not passed. Have you tried that?
> > 
> > 
> > @Nikhil-Ladha I made reverted the changes, but currently, the news page on the live site only directs to one URL even when you click on the 'see more' link. you can check it out. That was why I made changes to the CSS
> 
> What do you mean by one URL?
> There are 2 links in that card one is embedded to the whole card, that is wherever you click on the card it will redirect you to the news page within the layer5 site, except the 'new_page' icon, which redirects you to a separate page outside Layer5.
> I have confirmed the same.

@Nikhil-Ladha thanks for explaining better, I now understand and I have reverted the changes to the CSS. @leecalcote based on the way the Card component is it might be quite difficult to make the layer5.io link the one on the page unless I create a new object and change the current object. Should I do that?

PS: I apologize for late response, I was quite ill yesterday and couldn't do any work

**Comment by Nikhil-Ladha** [comment]
I think, that the only change which this PR need is changing the `false` value to `fields.slug`, and everything will be what is expected of it.
@Lezette could you please do that change and update the PR?

**Comment by Lezette** [comment]
> I think, that the only change which this PR need is changing the `false` value to `fields.slug`, and everything will be what is expected of it.
> @Lezette could you please do that change and update the PR?

Done

**Comment by l5io** [comment]
🚀 Preview for commit 586adb57f143f1aedaaf362214ab264d24266730 at: https://608862bd00f44900a6bd8f3e--layer5ng.netlify.app

**Comment by Nikhil-Ladha** [comment]
Ok, now I see there are some issues in the card UI.
Could you check if we are altering the height of card for this particular specifically? And tweak around that to get with this.

**Comment by Lezette** [comment]
@Nikhil-Ladha I the height was initially `8rem ` but when I made those changes I increased it to 9rem and then changed it back. but the height is only like that on the event page. I noticed it's cool on the news page

**Comment by Lezette** [comment]
@Nikhil-Ladha I added the field section but the event card still doesn't work as desired. Any idea on how to go about this?

**Comment by Nikhil-Ladha** [comment]
You need to tweak the CSS on this file https://github.com/layer5io/layer5/blob/e73934ddcc3e3c55fcad809c5806ceca662e67f6/src/sections/Events/meetups.style.js and not the actual component itself.
You can try and see if you get around it, else I will take a look and update here.

**Comment by l5io** [comment]
🚀 Preview for commit e73934ddcc3e3c55fcad809c5806ceca662e67f6 at: https://6088d3c7aead9254d6f9c327--layer5ng.netlify.app

**Comment by Nikhil-Ladha** [comment]
@Lezette I checked the code and was able to find the issue. Here are this things that could fix the issue:
- https://github.com/layer5io/layer5/blob/e73934ddcc3e3c55fcad809c5806ceca662e67f6/src/sections/Events/meetups.style.js#L63-L65  and  https://github.com/layer5io/layer5/blob/e73934ddcc3e3c55fcad809c5806ceca662e67f6/src/sections/Events/meetups.style.js#L71-L93 -> remove these lines of CSS, this will do the trick.

**Comment by Lezette** [comment]
Thank you. I’ll do that

**Comment by Lezette** [comment]
@Nikhil-Ladha thanks a lot, that fixed it. Also, my linter kept catching an incorrect code, `class` was used instead of `className` so I decided to add that also, is that alright?

File: src/components/Card/Card.style.js [hunk 2]
```diff
@@ -51,7 +51,7 @@ export const CardWrapper = styled.div`
 
     .post-content-block{
         padding: 1rem 2rem;
-        height: 8rem;
+        height: 9rem;
```
Comment (Nikhil-Ladha) [inline src/components/Card/Card.style.js:54]: This can be reverted.

File: src/sections/Company/Legal/privacy/index.js [hunk 3]
```diff
@@ -112,7 +112,7 @@ const Conduct = () => {
             </div>
             <p>We welcome your comments or questions about this privacy policy. You may also contact us as follows:
             </p>
-            <p class="address">
+            <p className="address">
```
Comment (Nikhil-Ladha) [inline src/sections/Company/Legal/privacy/index.js:115]: Ok. Can you checkout to the master branch and fetch the latest upstream, and check if confirm if we have this or not. I think it has been updated recently after you have opened this PR, so this change might no be needed.

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 71e2f2c065ab56234acf6a38a1d60253ac150271 at: https://608a267ace660f6dee240928--layer5ng.netlify.app

**Comment by Lezette** [comment]
Alright

File: src/sections/Company/Legal/privacy/index.js (hunk 3 above)
Comment (Lezette) [reply src/sections/Company/Legal/privacy/index.js:115]: I did that and there's no changes

Comment (Nikhil-Ladha) [reply src/sections/Company/Legal/privacy/index.js:115]: Interesting, I checked the master code and it's fixed.
Check this: https://github.com/layer5io/layer5/blob/7ef1455430b873b3c17ffc17654cf76cd34ca43b/src/sections/Company/Legal/privacy/index.js#L115

Comment (Nikhil-Ladha) [reply src/sections/Company/Legal/privacy/index.js:115]: Okay, let's ignore that for the time being considering there have been a lot of too and fro now, I will take that in and not de-motivate you to contribute more 😄

**Review by Nikhil-Ladha — APPROVED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 518600b6cd3b44a94f69812cc99e693d4fe6da1c at: https://608adcd1de8ea21b595d4f5b--layer5ng.netlify.app

**Pull request merged**